### DISCUSSION TITLE
ReVanced

### DISCUSSION THREAD

**Discussion by ItsProfessional** [catppuccin/catppuccin#2215] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

ReVanced is an open source patcher for android applications. It has a custom theme patch [for YouTube](https://github.com/ReVanced/revanced-patches/blob/main/src/main/kotlin/app/revanced/patches/youtube/layout/theme/ThemeBytecodePatch.kt) and [for Spotify](https://github.com/ReVanced/revanced-patches/blob/main/src/main/kotlin/app/revanced/patches/spotify/layout/theme/CustomThemePatch.kt) (as of writing).

The options (i.e. colors) for these patches can be configured using an `options.json` file when patching, see [here](https://github.com/ReVanced/revanced-cli/blob/main/docs/1_usage.md#%EF%B8%8F-generate-options) to generate one.

There's also a patch called [custom branding](https://github.com/ReVanced/revanced-patches/blob/main/src/main/kotlin/app/revanced/patches/youtube/layout/branding/CustomBrandingPatch.kt) to change the icon of the application to, for example, a catppuccin colored one.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by deviantsemicolon** [comment]
This would be cool
